### DISCUSSION TITLE
Default Errors - is there a way to do this easily?

### DISCUSSION THREAD

**Discussion by Hopp3r** [react-hook-form/react-hook-form#9460] [Q&A]
I'm creating a large wizard and storing values to a store and then setting them as default values when the form (wizard tab) is active again. That works well, but there's not really an easy solution for errors. 

Is there a simple way of setting default error state apart from parsing my store & calling setError for every field that previously had an error?

**Comment by bluebill1049** [comment]
Maybe try schema validation? it's all hosted in a single source.
